eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -75,26 +76,38 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
       statement.executeUpdate("CREATE INDEX IF NOT EXISTS relayed_payment_hash_idx ON relayed(payment_hash)")
     }
 
+    def migration45(statement: Statement): Int = {
+      statement.executeUpdate("ALTER TABLE relayed ADD recipientNodeId BLOB")
+      statement.executeUpdate("ALTER TABLE relayed ADD recipientAmount INTEGER")
```

Comment:
IIRC there was a limitation of SQLite that prevented us from simply doing an `ALTER TABLE` to add new columns, which is why we've done this weird dance in other migrations to re-create the table and copy old rows.

I'm not sure if it's a limitation on *all* versions of SQLite or only the one we use on mobile, but it's safer to do something similar to previous migrations.

Reply:
The limitation was that below 3.25.0, sqlite didn't allow column renaming. But we use 3.27.2.1 so we should be good.